config.json

```diff
@@ -638,19 +638,6 @@
       "unlocked_by": "rotational-cipher",
       "uuid": "d36ce010-210f-4e9a-9d6c-cb933e0a59af"
     },
-    {
```

Comment:
I think that this has already been merged in? You might be working on an old branch :slightly_smiling_face:

Reply:
How can this be fixed?